.github/CONTRIBUTING.md

```diff
@@ -95,11 +115,11 @@ git checkout -b your-branch-name
     
     - Provide concise explaination to the user about limitations of the current code, like a bug or an issue
 
-6. Examples of a great commit message:
+7. Examples of a great commit message:
 
     - Add CPU arch filter scheduler support
     
     - Fix submission worker crash issue
     
     - Change design template
-***Congratulations, you have successfully contributed to Project EvalAI!***
+***Congratulations, you have successfully contributed to Project EvalAI-ngx!***
```

Comment:
Ngx is only a temporary name until the frontend is merged with backend so . We should let it stay as Evalai